src/holofuel/components/RecentCounterparties/RecentCounterparties.js

```diff
@@ -2,15 +2,20 @@ import React from 'react'
 import { isEmpty } from 'lodash/fp'
 import cx from 'classnames'
 import HashIcon from 'holofuel/components/HashIcon'
+import Loading from 'components/Loading'
 import { presentAgentId } from 'utils'
 import './RecentCounterparties.module.css'
 
-export default function RecentCounterparties ({ agents = [], selectedAgentId, selectAgent, className }) {
+export default function RecentCounterparties ({ agents = [], selectedAgentId, selectAgent, className, loading }) {
```

Comment:
weird that these changes are in this PR. They should already be in develop.

Reply:
Agreed.